x-pack/plugins/security_solution/common/detection_engine/rule_management/model/import/rule_to_import.ts

```diff
@@ -44,6 +44,7 @@ export const RuleToImport = t.intersection([
       created_by,
       related_integrations: RelatedIntegrationArray,
       required_fields: RequiredFieldArray,
+      revision,
```

Comment:
What if we just ignore it and don't add to the schema? Will then a `ndjson` file with a `revision` be accepted, but the revision will be stripped out?